code/src/Form.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+
+import  NameInput from './components/NameInput'
+import  RadioButtons from './components/RadioButtons'
+import  DropDown  from './components/DropDown'
+import  Result  from './components/Result'
+
+const Form = () => {
+  const [name,setName]= useState('')
+  const [timeInterval, setTimeInterval]= useState('')
+  const [favorite, setFavorite]= useState('')
+  const [done, setDone]= useState(false)
+
+  const isResultComplete = () => {
+    if (name === '') {
+      return false
+    }
+
+    if (timeInterval === '') {
+      return false
+    }
+
+    if (favorite === '') {
+      return false
+    }
```

Comment:
I dont think you need this part since all the fields are required so its already checked at that point!